pkg/windows/windows.go

```diff
@@ -487,6 +495,33 @@ func (vm *windows) ConfigureWindowsExporter() error {
 	return nil
 }
 
+// ConfigureWICD starts the Windows Instance Config Daemon service
+func (vm *windows) ConfigureWICD(apiServerURL string, serviceAccountCA, serviceAccountToken []byte) error {
+	saCAFile := "sa-ca.crt"
+	saTokenFile := "sa-token"
+	err := vm.EnsureFileContent(serviceAccountCA, saCAFile, k8sDir)
+	if err != nil {
+		return err
+	}
+	err = vm.EnsureFileContent(serviceAccountToken, saTokenFile, k8sDir)
+	if err != nil {
+		return err
+	}
+	wicdPath := k8sDir + "windows-instance-config-daemon.exe"
```

Comment:
This can be moved with the other constants at the top of the file for consistency

Reply:
I'd prefer to not make this a global as this isnt used anywhere outside this function